shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -95,14 +95,14 @@
     public static final String CACHE_GET_LATENCY = "segmentstore.cache.get_latency_ms";         // Histogram
     public static final String CACHE_READ_BYTES = "segmentstore.cache.read_bytes";              // Counter
     public static final String CACHE_TOTAL_SIZE_BYTES = "segmentstore.cache.size_bytes";        // Gauge
-    public static final String CACHE_GENERATION_SPREAD = "segmentstore.cache.gen";              // Histogram
+    public static final String CACHE_GENERATION_SPREAD = "cache.gen";                           // Histogram
 
     // DurableDataLog (Tier1) stats
-    public static final String BK_TOTAL_WRITE_LATENCY = "segmentstore.bookkeeper.total_write_latency_ms";   // Including Queue. Per-container Histogram.
-    public static final String BK_WRITE_LATENCY = "segmentstore.bookkeeper.write_latency_ms";               // Exclusively the write to BK. Per-container Histogram.
+    public static final String BK_TOTAL_WRITE_LATENCY = "total_write_latency_ms";                           // Including Queue. Per-container Histogram.
+    public static final String BK_WRITE_LATENCY = "write_latency_ms";                                       // Exclusively the write to BK. Per-container Histogram.
     public static final String BK_WRITE_BYTES = "segmentstore.bookkeeper.write_bytes";                      // Counter
-    public static final String BK_WRITE_QUEUE_SIZE = "segmentstore.bookkeeper.write_queue_size";            // Per-container Histogram
-    public static final String BK_WRITE_QUEUE_FILL_RATE = "segmentstore.bookkeeper.write_queue_fill";       // Per-container Histogram
+    public static final String BK_WRITE_QUEUE_SIZE = "write_queue_size";                                    // Per-container Histogram
+    public static final String BK_WRITE_QUEUE_FILL_RATE = "write_queue_fill";                               // Per-container Histogram
     public static final String BK_LEDGER_COUNT = "segmentstore.bookkeeper.bookkeeper_ledger_count";         // Per-container Gauge
```

Comment:
This seems to be an overkill too (it works out to`pravega_segmentstore_bookkeeper_bookkeeper_ledger_count` in InfluxDB). 
Could you remove one of `bookkeeper` strings, please?

Reply:
fixed